src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -166,14 +170,8 @@ export const App = ({ loggedIn, toggleLoginModal, displayToggle }) => {
           <p>
             We’re sorry. Something went wrong when we tried to download your
             form. Please try again. If your form still doesn’t download, call us
-            at{' '}
-            <a href="tel:+18006982411" aria-label="1 8 0 0 6 9 8 2 4 1 1">
-              1-800-698-2411
-            </a>{' '}
-            (
-            <a href="tel:711" aria-label="TTY. 7 1 1">
-              TTY: 711
-            </a>
+            at <Telephone contact="8006982411" /> (TTY:{' '}
+            <Telephone contact={CONTACTS['711']} tty />
```

Comment:
This is the React component and I don't think it will work like you expect with the `tty` prop. The Web Component is what I'm recommending.

```suggestion
            <va-telephone contact={CONTACTS['711']} tty />
```